plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/internal/refresh/listeners/FontFormatUpdater.java

```diff
@@ -115,7 +118,35 @@ public Command transactionAboutToCommit(final ResourceSetChangeEvent event) thro
                 Style style = dDiagramElement.getStyle();
                 if (isViewFontStylePropertiesDifferentOfGMFOne(style, fontStyle, features)) {
                     Set<String> featureNames = getFeatureNames(features);
-                    if (!(style instanceof EdgeStyle)) {
+                    if (style instanceof EdgeStyle) {
```

Comment:
I would have added a comment to explain why this code is usefull and what (briefly) would be needed(the big dev that we do not want to do now) to get rid of it.